src/nupic/engine/Region.cpp

```diff
@@ -303,11 +303,12 @@ void Region::setDimensions(Dimensions &newDims) {
     dims_ = newDims;
     dimensionInfo_ = "Specified explicitly in setDimensions()";
   } else {
-    NTA_THROW << "Attempt to set dimensions of region " << getName() << " to "
+    NTA_THROW << "Attempt to set dimensions of region " << getName() << " to " 
               << newDims.toString() << " but region already has dimensions "
               << dims_.toString();
+    //NTA_THROW << "Attempt to set dimensions of region " << getName() << " to "
+    //          << newDims.toString() << " but region already initialized.";
```

Comment:
rm please

Reply:
> rm please

Hmmm, it looks like I have that feature half on and half off.  Sorry.  I will fix this